sru/release-28/azure_auto_attach_on_boot.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
+
+pro_deb=$1
+INSTANCE_NAME="pro-tat-vm"
+RESOURCE_GROUP="proclient-testing-auto-attach"
+LOCATION="centralus"
```

Comment:
this should not be fixed to `centralus`, because we still don't know in which region it will be available - better take it from the cli or some envvar